x/oracle/types/keys.go

```diff
@@ -30,61 +31,52 @@ var (
 	KeyPrefixHistoricPrice                = []byte{0x08} // prefix for each key to a historic price
 )
 
-// GetExchangeRateKey - stored by *denom*
-func GetExchangeRateKey(denom string) (key []byte) {
-	key = append(key, KeyPrefixExchangeRate...)
-	key = append(key, []byte(denom)...)
-	return append(key, 0) // append 0 for null-termination
+// KeyExchangeRate - stored by *denom*
+func KeyExchangeRate(denom string) []byte {
+	// append 0 for null-termination
+	return util.ConcatBytes(1, KeyPrefixExchangeRate, []byte(denom))
 }
 
-// GetFeederDelegationKey - stored by *Validator* address
-func GetFeederDelegationKey(v sdk.ValAddress) (key []byte) {
-	key = append(key, KeyPrefixFeederDelegation...)
-	return append(key, address.MustLengthPrefix(v)...)
+// KeyFeederDelegation - stored by *Validator* address
+func KeyFeederDelegation(v sdk.ValAddress) []byte {
+	// note: length prefix is not needed here!
```

Comment:
```suggestion
```

Reply:
I will create an issue instead of this comment.